### ISSUE TITLE
locale file not found

### ISSUE THREAD

**Issue by transtone** [e-chan1007/nuxt-monaco-editor#45]
```shell
  ERROR  Failed to resolve import "monaco-editor-nls/locale/zh-cn.json" from "node_modules\.pnpm\nuxt-monaco-editor@1.2.6_monaco-editor@0.45.0_vite@4.5.1\node_modules\nuxt-monaco-editor\dist\runtime\nls.mjs?v=a493b7b0". Does the file exist?
```
the doc need update. the right locale name is `zh-hans`.

==========

when use this settings:
```js
  monacoEditor: { locale: 'de' },
```

will get 

```

500
nls.localize2 is not a function

at new SelectToBracketAction (http://localhost:3000/_nuxt/node_modules/.pnpm/monaco-editor@0.45.0/node_modules/monaco-editor/esm/vs/editor/contrib/bracketMatching/browser/bracketMatching.js?v=05d667ad:47:34)
at registerEditorAction (http://localhost:3000/_nuxt/node_modules/.pnpm/monaco-editor@0.45.0/node_modules/monaco-editor/esm/vs/editor/browser/editorExtensions.js?v=05d667ad:318:20)
at http://localhost:3000/_nuxt/node_modules/.pnpm/monaco-editor@0.45.0/node_modules/monaco-editor/esm/vs/editor/contrib/bracketMatching/browser/bracketMatching.js?v=05d667ad:340:1

```

the `nls.localize2` error seems appear in `monaco-editor-0.45.0`, when goes back to version `0.39.0`, everythings ok.

**Comment by e-chan1007** [comment]
It should be fixed in `v1.2.9`. Please try it.

**Comment by transtone** [comment]
locale file is fixed.

but a new problem is: the context menu not closed when click the blank.
